Rakefile

```diff
@@ -8,7 +8,7 @@ ManualsPublisher::Application.load_tasks
 task default: [
   "lint",
   "assets:precompile",
```

Comment:
Sorry I know you didn't add this line, but I think we should remove it. I'm not sure why someone added it before but our change will make this completely redundant. As part of the jasmine yarn task we run `assets:clobber` which would delete any assets this task compiled.